tree/ntuple/v7/src/RNTupleReader.cxx

```diff
@@ -140,7 +162,8 @@ void ROOT::Experimental::RNTupleReader::PrintInfo(const ENTupleInfo what, std::o
       {
          auto descriptorGuard = fSource->GetSharedDescriptorGuard();
          name = descriptorGuard->GetName();
-         fullModel = descriptorGuard->CreateModel();
+         fullModel =
+            descriptorGuard->CreateModel(fCreateModelOptions.value_or(RNTupleDescriptor::RCreateModelOptions{}));
```

Comment:
Perhaps we want to go even a step further here: For printing we can always use emulated classes, so that you can always look at the dataset.

Reply:
Ok, I'll keep this in mind for the PR that adds the emulated classes